app/controllers/turfs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class TurfsController < ApplicationController
+  def create
+    unless current_user
+      flash[:notice] = "You must be signed in to do that."
+      render "/"
+      return
+    end
+    params["lists"].each_value do |list|
```

Comment:
what happens if the params["lists"] is left unspecified? Hint: it will be `nil`.